common/scripted_effects/wc_race_effects.txt

```diff
@@ -177,6 +177,16 @@ set_race_effect = {
 		limit = { scope:temp_race = flag:creature_kvaldir }
 		become_undead_effect = yes
 	}
+
+	if = { # Starts mana regeneration loop
+		limit = { NOT = { has_variable = potential_friend } }
```

Comment:
Huh? That's a vanilla term. What it's doing here?
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/120909302-d830ed80-c684-11eb-9d9c-8eec0a134e23.png)

</details>

Reply:
weird, don't remember it being there

Reply:
Probably forgot to replace it with variable below